config/samples/dataplane_v1beta1_openstackdataplane_nmstate.yaml

```diff
@@ -0,0 +1,258 @@
+                ipv4:
+                  enabled: true
+                  address:
+                    - ip: "172.19.0.101"
+                      prefix-length: 24
+              - name: br-ex
+                type: ovs-bridge
+                bridge:
+                  options:
+                    fail-mode: standalone
+                  port:
+                  - name: eth0
+                  - name: br-ex
+                  - name: vlan20
+                    vlan:
+                      mode: access
+                      tag: 20
+                  - name: vlan21
+                    vlan:
+                      mode: access
+                      tag: 21
+                  - name: vlan22
+                    vlan:
+                      mode: access
+                      tag: 22
+            routes:
+              config:
+              - destination: 0.0.0.0/0
+                next-hop-address: 192.168.122.1
+                next-hop-interface: br-ex
+        ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+      deployStrategy:
+        deploy: false
+  roles:
+    edpm-compute:
+      env:
+        - name: ANSIBLE_FORCE_COLOR
+          value: "True"
+        - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+          value: "True"
+        - name: ANSIBLE_VERBOSITY
+          value: "2"
+      baremetalSetTemplate:
+        deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
+        ctlplaneInterface: enp1s0
+        provisionServerName: openstackprovisionserver
+        ctlplaneGateway: 172.22.0.3
+        ctlplaneNetmask: 255.255.255.0
+        bmhNamespace: openstack
+        domainName: osptest.openstack.org
+      preProvisioned: true
+      services:
+        - configure-network
+        - validate-network
+        - install-os
+        - configure-os
+        - run-os
+      nodeTemplate:
+        # Defining the novaTemplate here means the nodes in this role are
+        # computes
+        nova: {}
+        managementNetwork: ctlplane
+        ansibleUser: root
+        ansiblePort: 22
+        ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+        ansibleVars: |
+          service_net_map:
+            nova_api_network: internal_api
+            nova_libvirt_network: internal_api
+
+          # edpm_nodes_validation
+          edpm_nodes_validation_validate_controllers_icmp: false
+          edpm_nodes_validation_validate_gateway_icmp: false
+
+          edpm_ovn_metadata_agent_DEFAULT_transport_url: rabbit://default_user@rabbitmq.openstack.svc:5672
+          edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: tcp:10.217.5.121:6642
+          edpm_ovn_metadata_agent_metadata_agent_DEFAULT_nova_metadata_host: 127.0.0.1
+          edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: 12345678
+          edpm_ovn_metadata_agent_DEFAULT_bind_host: 127.0.0.1
+          edpm_chrony_ntp_servers:
+          - clock.redhat.com
+          - clock2.redhat.com
```

Comment:
should those be public ones for the sample?

Reply:
Probably, but this is what we have in our other samples, so I think we can clean it up across the board separately.